opt_einsum/backends/torch.py

```diff
@@ -39,8 +45,6 @@ def tensordot(x, y, axes=2):
     """Simple translation of tensordot syntax to einsum.
     """
     # XXX: tensordot should be directly implemented in torch soon
-    torch, _ = _get_torch_and_device()
```

Comment:
@jcmgray Can you look over this as well?

Reply:
(Note the only reason I've removed this line is so I can reuse the `einsum()` wrapper in this file, rather than calling `torch.einsum` in two different places)